### PR TITLE
Added dummy get last index in Exec

### PR DESCRIPTION
**Pull request by kisbg**
The realted test262 test-cases has been removed from skip list

Signed-off-by: bence gabor kis <kisbg@inf.u-szeged.hu>

### CODE REVIEW

File: tools/test/v8/v8.mjsunit.status [hunk 1]
```diff
@@ -37,6 +37,7 @@
   ##############################################################################
   # Fails.
   'regress/regress-1119': [FAIL],
+  'regress/regress-2438': [FAIL],
```
Comment (clover2123) [inline tools/test/v8/v8.mjsunit.status:40]: Why is this TC failed?

Comment (kisbg) [reply tools/test/v8/v8.mjsunit.status:40]: Sadly it's an outdated test case.

Comment (clover2123) [reply tools/test/v8/v8.mjsunit.status:40]: I see. LGTM

Comment (kisbg) [reply tools/test/v8/v8.mjsunit.status:40]: Thank you!

**Pull request merged**